process/economics/economicsData_test.go

```diff
@@ -33,9 +33,11 @@ func createDummyEconomicsConfig() *config.ConfigEconomics {
 			BurnPercentage:      0.8,
 		},
 		FeeSettings: config.FeeSettings{
-			MaxGasLimitPerBlock: "100000",
-			MinGasPrice:         "18446744073709551615",
-			MinGasLimit:         "500",
+			MaxGasLimitPerBlock:  "100000",
+			MinGasPrice:          "18446744073709551615",
```

Comment:
Does this value have a meaning? Why not 1 and a lot of zeroes?

Reply:
no meaning in the unit tests.

Reply:
ok